sycl/include/CL/sycl/swizzles.def

```diff
@@ -9,833 +9,804 @@
 // Included by types.hpp twice, once for vec<> and once for SwizzleOp<>.
 
 // TODO: exclude L-Value swizzle like vec.xxxx()
-#ifdef __SYCL_ACCESS
-#error Undefine __SYCL_ACCESS macro.
+
+#if defined(__SYCL_ACCESS) || defined(__SYCL_INDEXER) || \
+    defined __SYCL_EXPAND || defined(__SYCL_NTH_ARG) || \
```

Comment:
defined __SYCL_EXPAND -> defined(__SYCL_EXPAND)

Reply:
Fixed. Thank you.